network-api/networkapi/wagtailpages/models.py

```diff
@@ -848,6 +854,8 @@ class BlogPage(FoundationMetadataPageMixin, Page):
     def get_context(self, request):
         context = super().get_context(request)
         context['related_posts'] = get_content_related_by_tag(self)
+        context['coralTalkServerUrl'] = settings.CORAL_TALK_SERVER_URL
```

Comment:
Let's stick with python conventions here.
```suggestion
        context['coral_talk_server_url'] = settings.CORAL_TALK_SERVER_URL
```